package.json

```diff
@@ -14,36 +14,33 @@
     "type": "git",
     "url": "git://github.com/mongoosastic/mongoosastic"
   },
-  "main": "lib/mongoosastic.js",
+  "main": "lib/index.ts",
```

Comment:
this will be in the build? dist/index.js? or how this is built and used?

Reply:
I tried to make sure that the code & tests are working, i will add the build script and create a commit, i will add .npmignore as well.